spec/lib/rwj_reporter_spec.rb

```diff
@@ -102,19 +84,16 @@
 
     context '#print_channel_counts_files' do
       before(:context) do
-        RwjReporter.print_reports_for_dates('160228', '160303', FIXTURE_DIR, OUTPUT_DIR)
+        RwjReporter.new('160228', '160303', FIXTURE_DIR).print_reports_for_dates
       end
       let(:channel1_data_array) { File.open(File.join(OUTPUT_DIR, '160228-160303_channel_1_usage_counts.csv')).readlines }
       let(:channel2_data_array) { File.open(File.join(OUTPUT_DIR, '160228-160303_channel_2_usage_counts.csv')).readlines }
-      it 'each channel file is sorted by show number' do
-        expect(channel1_data_array).to eq channel1_data_array.sort
-        expect(channel2_data_array).to eq channel2_data_array.sort
-      end
```

Comment:
If we are still having channel files, should we still be ensuring they are sorted by show number?  Not clear to me why this test went away.